src/dev/so_migration/so_migration_compare_cli.ts

```diff
@@ -35,14 +35,17 @@ run(
       '--from <rev|filename|url>',
       '--to <rev|filename|url>',
       '[--outputPath <outputPath>]',
+      '[--short]',
     ].join(' '),
     description: `Compares two Saved Object snapshot files based on hashes, filenames or urls.`,
     flags: {
+      boolean: ['short'],
       string: ['outputPath', 'from', 'to'],
       help: `
         --from            The source snapshot to compare from. Can be a revision, filename or url.
         --to              The target snapshot to compare to. Can be a revision, filename or url.
         --outputPath      The path to write the comparison report to. If omitted, raw JSON will be output to stdout.
+        --short           If true, only the list of changed plugins will be output.
```

Comment:
Maybe it would be more "standard" to make this the default behavior and invert the flag to be `verbose`?

Reply:
I also feel a bit off, because it's not often that you request for shorter output. 
However, the current main use case is for parsing the JSON output, where we don't want this kind of output exactly for the parseability, that's why I approached it from the other angle. Maybe it's up for some more finetuning :)

Just to recap, there are 2 output items: 
- (1) human-readable logs and summary
- (2) machine parseable JSON output

Currently:
- no arguments yield (1) + (2)
- `--short` or `--outputPath` will yield (1) only
- `--quiet` will yield (2) only

Maybe it's naive to think we'll ever need (1)+(2) together because if a human wants to read it, they probably don't care for the long JSON. 

So I could change it to have a flag: `--json`, with it we get (2), without it we get (1) only

Reply:
Current solution:
- no args: outputs (1) only
- `--json`: outputs (1) and (2)
- `--json --quiet`: outputs (2) only

Reply:
Thanks for investigating, this seems reasonable given the restrictions.